### ISSUE TITLE
[Enhancement] Pouch takes a .webhapp instad of a .happ + ui.zip

### ISSUE THREAD

**Issue by mattyg** [holochain-apps/holochain-kangaroo#9]
Does it make sense for the pouch to hold a .webhapp, instead of both a .happ and a ui.zip? Seems simpler, right?

**Comment by matthme** [comment]
Absolutely, I actually wanted to start implementing this recently but ended up not prioritizing it. One thing that held me off from implementing it is that unpacking a webhapp is easiest via using the `hc` CLI and this would require `hc` to be present in the environment, i.e. probably nix shell. I guess that's acceptable but it seemed like an additional point of friction as well.

**Comment by matthme** [comment]
Note that the unpacking needs to happen at build time, not runtime. But it might still be possible to use it in a build.rs file, will have to look into it.
